### PR TITLE
Remove django objects

### PR DESCRIPTION
**Pull request by rafalp**
This PR replaces `DjangoObjectType` and `CountableDjangoObject` type in our schema with `ModelObjectType` that brings no special logic or magic with it, but it knows that it represents Django Model, so we can still scan schema's types for GraphQL types supposed to represent given model.

All fields are now explicitly declared on GraphQL types (vs being populated automagically by `only_fields`.

Reliance on global type -> model registry is also removed by this change. I'm not 100% sure because Graphene has it's own types registry I've haven't investigated in detail, but it looks like this brings us step closer to splitting our GraphQL schema into Admin and Public schema.

There's still few places that import different utils from `graphene_django` packages (mostly for filtering), those will need to be extracted to Saleor in next PR.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by korycins — APPROVED** [review]

File: saleor/graphql/translations/schema.py [hunk 1]
```diff
@@ -43,6 +43,33 @@ class Meta:
...
+        if isinstance(instance, ShippingMethod):
+            return translation_types.ShippingMethodTranslatableContent
+        if isinstance(instance, Sale):
+            return translation_types.SaleTranslatableContent
+        if isinstance(instance, Voucher):
+            return translation_types.VoucherTranslatableContent
+        if isinstance(instance, MenuItem):
+            return translation_types.MenuItemTranslatableContent
+
+        return super(TranslatableItem, cls).resolve_type(instance, info)
```
Comment (fowczarek) [inline saleor/graphql/translations/schema.py:71]: Maybe we could create a map and simplify this function?

**Review by fowczarek — APPROVED** [review]
Missing changelog :)

File: saleor/graphql/translations/schema.py (hunk 1 above)
Comment (rafalp) [reply saleor/graphql/translations/schema.py]: @fowczarek I've added mapping.

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/account/types.py [hunk 2]
```diff
@@ -55,10 +50,21 @@ class AddressInput(graphene.InputObjectType):
 
 
 @key(fields="id")
-class Address(CountableDjangoObjectType):
+class Address(ModelObjectType):
```
[the remark this answers is not in the document]
Comment (maarcingebala) [reply saleor/graphql/account/types.py:53]: I think Owczar meant descriptions in the below fields - almost all of them are missing it.

**Review by maarcingebala — APPROVED** [review]

**Comment by rafalp** [comment]
@maarcingebala don't worry about that, task is up already ;)

**Comment by maarcingebala** [comment]
These changes look great but I have one concern - if we merged it today it would potentially make any ports from 3.0 to 3.1 more difficult, as all types are changed here. I'm wondering if we shouldn't wait with merging this one until stable 3.0 is released. Will discuss it with @fowczarek and @korycins.

**Pull request merged**